queue/queue.c

```diff
@@ -2,7 +2,8 @@
 #include <stdlib.h>
 #include "lib.h"
 
-typedef struct Queue {
+typedef struct Queue
+{
```

Comment:
I like your formatting here, it's very similar to Objective C or C sharp. They have a very easy to read common format like this.